docs/introduction/how-to-work-with-money.md

```diff
@@ -0,0 +1,388 @@
+- `Money::isLessThanOrEqualTo(Money $other) : bool` instead of `<=`
+- `Money::isGreaterThanOrEqualTo(Money $other) : bool` instead of `>=`
+- `Money::compare(Money $other) : int` instead of the Spaceship operator `<=>`
+
+To compare a value with zero you may use the short-hand methods`isPositive`, `isNegative` or `isZero`.
+
+All methods compare the amounts of both objects counting all decimal places.
+
+## Money in Forms
+
+For the user input of monetary values use the `MoneyType` (see [Symfony docs](https://symfony.com/doc/3.4/reference/forms/types/money.html) for details).
+
+```php
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+// ...
+
+$orderItemFormBuilder->add('priceWithVat', MoneyType::class, [
+    'scale' => 6,
+    'constraints' => [
+        new NotBlank(['message' => 'Please enter unit price with VAT']),
+    ],
+]);
+```
+
+The form type is configured with a model data transformer that converts the value into a `Money` object automatically (`\Shopsys\FrameworkBundle\Form\Transformers\NumericToMoneyTransformer`).
+
+In Shopsys Framework, the default value of the `currency` option is `false` instead of `EUR`, hiding the currency symbol by default.
+
+*Note: For non-monetary numeric values use `NumberType` (see [Symfony docs](https://symfony.com/doc/3.4/reference/forms/types/number.html) for details).*
+
+### Form Constraints
+
+There are two form constraints to be used specifically with the `MoneyType` form fields.
+Because of model data transformation, you cannot use constraints that are validating scalar values (eg. `GreaterThan`).
+
+#### NotNegativeMoneyAmount
+
+Validates that the amount of money is greater or equal to zero.
+
+It has only the `message` option specifying the validation error message in case the entered value is negative.
+
+```php
+use Shopsys\FrameworkBundle\Form\Constraints\NotNegativeMoneyAmount;
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+// ...
+
+$priceTableFormBuilder->add($key, MoneyType::class, [
+    'scale' => 6,
+    'required' => true,
+    'invalid_message' => 'Please enter price in correct format (a number with decimal separator)',
+    'constraints' => [
+        new NotBlank(['message' => 'Please enter price']),
+        new NotNegativeMoneyAmount(['message' => 'Price must be greater or equal to zero']),
+    ],
+]);
+```
+
+#### MoneyRange
+
+Similarly to [the Symfony `Range` constraint](https://symfony.com/doc/3.4/reference/constraints/Range.html), it validates that the amount of money is between some minimum and maximum.
+
+It has four options:
+- `min` specifies the minimum value, has to be an instance of `Money` or `null`
+- `max` specifies the maximum value, has to be an instance of `Money` or `null`
+- `minMessage` specifies the validation error message in case the entered value is less than the `min` value
```

Comment:
Please clarify these options are optional and they have default value

Reply:
Addressed in 4664094d0